frontend/src/cljs/imas_seamap/map/utils.cljs

```diff
@@ -219,13 +222,32 @@
         properties (map (fn [[label value]] {:label label :value value}) (get-in parsed ["features" 0 "properties"]))
         property-to-row (fn [{:keys [label value]}] (str "<tr><td>" label "</td><td>" value "</td></tr>"))
         property-rows (str/join "" (map (fn [property] (property-to-row property)) properties))]
-    (if (or id (not-empty properties))
-      {:info (str
-              "<div class=\"feature-info-json\">"
-              "<h4>" id "</h4>"
-              "<table>" property-rows "</table>"
-              "</div>")}
-      {:status :feature-info/empty})))
+    (when (or id (not-empty properties))
+      {:style
+       (str
+        ".feature-info-json {"
+        "    max-height: 257px;"
+        "    overflow-y: scroll;"
+        "}"
```

Comment:
Don't know why I didn't do it myself, but worth noting that clojure has multi-line strings out of the box.

Reply:
Yeah, I noticed that, but it has mucked up indentation, at least in my editor. I'll use them in future, knowing they are preferred.